velox/expression/FunctionSignature.h

```diff
@@ -24,29 +24,47 @@ namespace facebook::velox::exec {
 
 std::string sanitizeFunctionName(const std::string& name);
 
+inline bool isCommonDecimalName(const std::string& typeName) {
+  return (typeName == "DECIMAL");
+}
+
 // A type name (e.g. K or V in map(K, V)) and optionally constraints, e.g.
-// orderable, sortable, etc.
+// expression over the variable, etc.
 class TypeVariableConstraint {
  public:
   explicit TypeVariableConstraint(std::string name) : name_{std::move(name)} {}
+  TypeVariableConstraint(std::string name, std::string constraint)
```

Comment:
Is constraint optional? Would it make sense to use std::optional<std::string>? Otherwise, perhaps, document that empty string is used if there is no constraint.